bc_obps/common/permissions.py

```diff
@@ -0,0 +1,81 @@
+from typing import Callable, Dict, Literal
+from django.http import HttpRequest
+from registration.utils import raise_401_if_user_not_authorized
+from registration.models import AppRole, UserOperator
+
+all_authorized_app_roles = AppRole.get_all_authorized_app_roles()
+all_industry_user_operator_roles = UserOperator.get_all_industry_user_operator_roles()
+
+permission_configs: Dict[str, Dict] = {
+    "everyone": {
+        'authorized_app_roles': AppRole.get_all_app_roles(),
+        'authorized_user_operator_roles': all_industry_user_operator_roles,
+        'industry_user_must_be_approved': False,
+    },
+    "authorized_roles": {
+        'authorized_app_roles': all_authorized_app_roles,
+        'authorized_user_operator_roles': all_industry_user_operator_roles,
+        'industry_user_must_be_approved': False,
+    },
+    "approved_industry_user": {
+        'authorized_app_roles': ["industry_user"],
+        'authorized_user_operator_roles': all_industry_user_operator_roles,
+    },
+    "industry_user_only": {
```

Comment:
```suggestion
    "industry_user": {
```

Reply:
suggestion for consistency; `only` isn't used in the longer names like `approved_industry_admin_user` even though it's also only industry, not cas